src/gulp-jsdoc.js

```diff
@@ -81,8 +89,6 @@ export function jsdoc(config, done) {
                     args = args.concat(['-t', inkdocstrap]);
                 }
 
-                args = args.concat(files);
```

Comment:
If that's what you want to do, it would make more sense to just append the sources to the `include` array. I want to avoid passing via command-line entirely.

I still really think that this should really be a one or the other scenario, but that's up to you.